pkg/crypto/keys/publickey_test.go

```diff
@@ -57,7 +75,27 @@ func TestDecodeBytes(t *testing.T) {
 	decodedPubKey := &PublicKey{}
 	err := decodedPubKey.DecodeBytes(pubKey.Bytes())
 	require.NoError(t, err)
-	require.Equal(t, pubKey,decodedPubKey)
+	require.Equal(t, pubKey, decodedPubKey)
+}
+
+func TestSort(t *testing.T) {
+	unsorted := make(PublicKeys, 10)
+	for i := range unsorted {
+		priv, err := NewPrivateKey()
+		require.NoError(t, err)
+		unsorted[i] = priv.PublicKey()
+	}
+
+	pubs := make(PublicKeys, len(unsorted))
+	copy(pubs, unsorted)
+
+	sort.Sort(pubs[:])
+	require.NotEqual(t, unsorted, pubs)
+
+	// check if sort is stable
```

Comment:
That's some kind of a different definition of stable sorting than I'm used to. I don't think we care about sorting stability (as in equal elements ordering preservation) here and if you're checking for something else than the comment should say what are you really checking here.